islandora_videojs.module

```diff
@@ -70,6 +70,11 @@ function template_preprocess_islandora_videojs(array &$variables) {
   elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  $variables['center_play_button'] = (variable_get('islandora_videojs_center_play_button');
+  $variables['responsive'] = (variable_get('islandora_videojs_responsive');
```

Comment:
Hi!, i see ")" missing here and on line 73 wich throw logic erros. See https://travis-ci.org/Islandora/islandora_videojs/jobs/301415544#L927-L933